docs/standards/smart-contracts/lsp1-universal-receiver-delegate-vault.md

```diff
@@ -3,18 +3,18 @@ title: LSP1 Universal Receiver Delegate Vault
 sidebar_position: 9
 ---
 
-# LSP1UniversalReceiverDelegateVault
+# LSP1 Universal Receiver Delegate Vault
```

Comment:
Please revert

The name should stay as it was before, it is the name of the smart contract